recipes/range-v3/bld.bat

```diff
@@ -0,0 +1,20 @@
+mkdir build
+cd build
+
+cmake -G "Visual Studio 16 2019" -A x64 ^
```

Comment:
```suggestion
cmake ^
```

Reply:
Ahh so you meant to complete remove the ``-G`` option?